src/model/jwks.rs

```diff
@@ -164,15 +179,56 @@ impl Jwk {
     }
 }
 
+fn generate_x509_cert(key_pair: &PKey<Private>) -> Result<X509, ErrorStack> {
+    let mut x509_name = X509NameBuilder::new()?;
+    x509_name.append_entry_by_text("C", "IT")?;
```

Comment:
I would add some configurations for this. This is a bit too much 'prima.it'-specific

Reply:
With this i mean to not hardcode those values but put them in the localauth0.toml file (with a default set to values that do not implies prima.it)

Reply:
It seems that these parameters are not necessary, we decided to remove them.

Reply:
Maybe keeping something recognizable as LocalAuth0 could be useful for debugging in the future?

Reply:
Added again (hardcoded).